q/Q.d.ts

```diff
@@ -15,17 +15,17 @@ declare function Q<T>(value: T): Q.Promise<T>;
 /**
  * Calling with nothing at all creates a void promise
  */
-declare function Q(): Q.Promise<void>;
+declare function Q<T>(): Q.Promise<void>;
```

Comment:
what does this `T` mean? it is not inferred from anywhere, nor is it used anywhere.